scheduler/src/cook/mesos.clj

```diff
@@ -178,14 +178,17 @@
    mea-culpa-failure-limit  -- long, max failures of mea culpa reason before it is considered a 'real' failure
                                      see scheduler/docs/configuration.asc for more details
    task-constraints         -- map, constraints on task. See scheduler/docs/configuration.asc for more details
+   executor                 -- cook executor config includes command, log-level, message-length, progress-output-file,
+                                     progress-regex-string, progress-sample-interval-ms and uri.
    riemann-host             -- str, dns name of riemann
    riemann-port             -- int, port for riemann
    mesos-pending-jobs-atom  -- atom, Populate (and update) list of pending jobs into atom
    offer-cache              -- atom, map from host to most recent offer. Used to get attributes
    gpu-enabled?             -- boolean, whether cook will schedule gpus
    rebalancer-config        -- map, config for rebalancer. See scheduler/docs/rebalancer-config.asc for details
    fenzo-config             -- map, config for fenzo, See scheduler/docs/configuration.asc for more details"
-  [make-mesos-driver-fn get-mesos-utilization curator-framework mesos-datomic-conn mesos-datomic-mult zk-prefix offer-incubate-time-ms mea-culpa-failure-limit task-constraints riemann-host riemann-port mesos-pending-jobs-atom offer-cache gpu-enabled? rebalancer-config mesos-leadership-atom
+  [make-mesos-driver-fn get-mesos-utilization curator-framework mesos-datomic-conn mesos-datomic-mult zk-prefix offer-incubate-time-ms
+   mea-culpa-failure-limit task-constraints executor riemann-host riemann-port mesos-pending-jobs-atom offer-cache gpu-enabled? rebalancer-config mesos-leadership-atom
```

Comment:
`executor` was added. Where would you prefer to place the parameter?

organizing: I broke it up across multiple lines as @sradack requested it.